doc/source/text.rst

```diff
@@ -429,6 +429,27 @@ String ``Index`` also supports ``get_dummies`` which returns a ``MultiIndex``.
 
 See also :func:`~pandas.get_dummies`.
 
+Concatenating Series
+--------------------
+
+The method :meth:`Series.str.cat` can be used to concatenate the records of two :class:`Series`. Depending on the value given to the ``align`` keyword,
+their indexes will be aligned before concatenation (if ``align=True``) or not (if ``align=False``). As usual, alignment will expand to the union of both
+indexes, while introducing ``NaN`` for missing values in the respective other series (which can be easily handled with the ``na_rep``-keyword).
+
+.. warning::
```

Comment:
you don't need this you are already passing the align keyword. These docs should be written as if a user is seeing them w/o benefit of any past history. Just show what they *should* do.

Reply:
Do you mean the warning? The text was verbatim from @TomAugspurger, but I will rework this as well.

Reply:
its not necessary here is my point.